metadata/src/main/java/org/apache/kafka/controller/metrics/QuorumControllerMetrics.java

```diff
@@ -267,7 +267,8 @@ public long newActiveControllers() {
     }
 
     public void updateBrokerContactTime(int brokerId) {
-        brokerContactTimesMs.putIfAbsent(brokerId, new AtomicLong(time.milliseconds()));
```

Comment:
Excuse me, I don't understand the point of using `AtomicLong`. I check the usage of `brokerContactTimesMs` and it seems `ConcurrentHashMap<Integer, Long>` would be sufficient, right?